### DISCUSSION TITLE
win11双击vse.exe后没有任何反应。无论是win版还是win-cpu版

### DISCUSSION THREAD

**Discussion by Naozheng** [YaoFANGUK/video-subtitle-extractor#196] [General]
如题，是因为win11的缘故么？

**Comment by YaoFANGUK** [comment]
开启的时候有点慢，如果等半天都不行，发一下电脑的配置和运行截图，协助排查一下

**Comment by kevinliu616** [comment]
請問win cpu版 執行檔位置在哪裡啊？ 找不到執行檔名是哪個？請告知一下 謝謝 解壓後沒有vse.exe啊

**Reply by YaoFANGUK** [reply]
github的release里面下载一下压缩包

**Reply by kevinliu616** [reply]
> github的release里面下载一下压缩包

下載了 執行檔是哪個？ gui.exe嗎 
win11 執行gui.exe沒反應....

**Reply by Wings1106** [reply]
同问，win11执行exe没有反应
